artemis-server/src/main/java/org/apache/activemq/artemis/core/server/protocol/websocket/WebSocketServerHandler.java

```diff
@@ -117,7 +118,8 @@ private boolean handleWebSocketFrame(ChannelHandlerContext ctx, WebSocketFrame f
       } else if (frame instanceof PingWebSocketFrame) {
          ctx.writeAndFlush(new PongWebSocketFrame(frame.content().retain()));
          return false;
-      } else if (!(frame instanceof TextWebSocketFrame) && !(frame instanceof BinaryWebSocketFrame)) {
+      } else if (!(frame instanceof TextWebSocketFrame) && !(frame instanceof BinaryWebSocketFrame) &&
```

Comment:
small change! zero test on a core hot path functionality!!! Really dangerous to merge this man!

Can you provide tests for this? or some background on why a test is not needed? (would this be covered by existing tests? which I don't believe it's the case).

Reply:
I will close this for now: 

you would need to provide:

- A JIRA
- a test as part of your PR
- change your commit name to include the JIRA name.


I will have to close it, as this is not even close to something I would be able to merge.

Reply:
Thank you I will create the Jira ticket etc.